src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -49,6 +49,7 @@ var registrationNameModules = ReactBrowserEventEmitter.registrationNameModules;
 
 // For quickly matching children type, to test if can be treated as content.
 var CONTENT_TYPES = {'string': true, 'number': true};
+var CHILDREN = keyOf({children: null});
 
 var STYLE = keyOf({style: null});
```

Comment:
nit: Want to put `CHILDREN` with `STYLE`. And actually if you rebase, you'll pick up the new one here also using keyof, would be good to have the group of them together